REED_WIFI/REED_WIFI.ino

```diff
@@ -0,0 +1,281 @@
+
+//#include <ESP8266WiFi.h>
+#include <ArduinoOTA.h>
```

Comment:
**General remark**
- Remove all comments that are not needed.
- Remove unnecessary println.
- Use only english language in code.